vmdb/app/models/ems_refresh/parsers/openstack_common/orchestration_stacks.rb

```diff
@@ -0,0 +1,186 @@
+module EmsRefresh
+  module Parsers
+    module OpenstackCommon
+      module OrchestrationStacks
+        def stack_resources(stack)
+          return @resources[stack.id] if @resources && !@resources.fetch_path(stack.id).blank?
+          @resources = {} unless @resources
+          # TODO(lsmola) catch exception when nested-depth is not supported, it's supported from Juno OpenStack
+          @resources[stack.id] = @orchestration_service.list_resources(stack, :nested_depth => 50).body['resources']
+        end
+
+        def load_orchestration_stacks
+          process_collection(stacks, :orchestration_stacks) { |stack| parse_stack(stack) }
+          update_nested_stack_relations
+        end
+
+        private
+
+        def stacks
+          @stacks ||= detailed_stacks
+        end
+
+        def detailed_stacks
+          # Seems like Heat doesn't support detailed list, so we need to call get for each stack, which is suboptimal
+          @orchestration_service.stacks.collect(&:details)
```

Comment:
@ladas, can you file an RFE upstream for Heat?  Seems like something that should be easy to add to their API and would save a lot of calls here.  We won't wait on the RFE, but would be nice to have it filed.